##########################################################
##########     Object Information(goal)   ######################

uint32 object_id
string object_name

geometry_msgs/PoseStamped object_pose

string grasp_database
uint32 grasp_id

string support_surface

---
##########################################################
##########     Pickup Result (result)   ######################

std_msgs/Bool success

string cob_pick_error_string

---

##########################################################
##########     Pickup Feedback (feedback)   ######################

std_msgs/Bool success

string cob_pick_error_string
